src/libmowgli/platform/machine.h

```diff
@@ -173,6 +173,11 @@
 #  define MOWGLI_CPU superh
 #  define MOWGLI_CPU_BITS_32
 #  define MOWGLI_CPU_BITS 32
+#elif defined __loongarch__ || defined __loongarch64
```

Comment:
The CPU Bits on loongarch32 is not 64, so this line use ```#elif defined (__loongarch64)``` will be better.

Reply:
> The CPU Bits on loongarch32 is not 64, so this line use `#elif defined (__loongarch64)` will be better.

thank you ,I will change